src/docker/start.sh

```diff
@@ -81,6 +81,21 @@ else
   print_header "No custom SSL certificate provided"
 fi
 
+cleanup() {
+  if [ -e config.sh ]; then
```

Comment:
Why test the presence of `config.sh`? The file is not tested anywhere else in the script.